webapp/messages/en.json

```
@@ -224,8 +224,11 @@
         "tvl": "TVL (on Hemi)",
         "website": "Website"
       },
+      "unstake-eth-as-weth-heading": "Unstaked ETH will be received as WETH",
+      "unstake-eth-as-weth-subheading": "When you stake ETH, Hemi automatically wraps it as WETH. Upon unstaking, you’ll receive WETH, which you can unwrap back to ETH if desired.",
```

Hemi does not do that. Can we just say something like...

```suggestion
      "unstake-eth-as-weth-subheading": "When you stake ETH, it is automatically converted to WETH. Upon unstaking, you’ll receive WETH, which you can unwrap back to ETH if desired.",
```

---

Updated in [b5ba553](https://github.com/hemilabs/ui-monorepo/pull/968/commits/b5ba553cb45e4d4ff53761c57cc63825c11dd572) - See if it now looks better